rtgui/preferences.cc

```diff
@@ -509,14 +509,14 @@ Gtk::Widget* Preferences::getProcParamsPanel ()
     Gtk::HBox* hb42 = Gtk::manage (new Gtk::HBox ());
     darkFrameDir = Gtk::manage (new Gtk::FileChooserButton (M ("PREFERENCES_DIRDARKFRAMES"), Gtk::FILE_CHOOSER_ACTION_SELECT_FOLDER));
     Gtk::Label *dfLab = Gtk::manage (new Gtk::Label (M ("PREFERENCES_DIRDARKFRAMES") + ":"));
-    hb42->pack_start (*dfLab , Gtk::PACK_SHRINK, 4 );
+    hb42->pack_start (*dfLab, Gtk::PACK_SHRINK, 4 );
     hb42->pack_start (*darkFrameDir, Gtk::PACK_EXPAND_WIDGET, 4);
     dfLabel = Gtk::manage (new Gtk::Label ("Found:"));
     Gtk::VBox* vbdf = Gtk::manage (new Gtk::VBox ());
     vbdf->pack_start ( *hb42, Gtk::PACK_SHRINK, 4);
     vbdf->pack_start ( *dfLabel, Gtk::PACK_SHRINK, 4 );
     fdf->add ( *vbdf );
-    mvbpp->pack_start ( *fdf , Gtk::PACK_SHRINK, 4);
+    mvbpp->pack_start ( *fdf, Gtk::PACK_SHRINK, 4);
```

Comment:
@Beep6581 I played a bit with astyle 3 and this file. The only option to remove spaces (inside the brace `mvbpp->pack_start ( *fdf, Gtk::PACK_SHRINK, 4)`) is `unpad-paren`. When adding it to `rawtherapee.astylerc` the problem with `pad-first-paren-out` becomes apparent: Empty call operators are rewritten as `func()` whereas those with arguments become `func (arg)`. So I ended up with:

```
style=1tbs
indent=spaces=4
convert-tabs
indent-switches
break-blocks
unpad-paren
pad-oper
#pad-first-paren-out
pad-header
```

Seems that astyle isn't prepared for the "space in front of call" case (which is rare, IMHO).

HTH,
Flössie